system/Cookie/CloneableInterface.php

```diff
@@ -9,15 +9,15 @@
  * file that was distributed with this source code.
  */
 
-namespace CodeIgniter\HTTP\Cookie;
+namespace CodeIgniter\Cookie;
 
 use DateTimeInterface;
 
 /**
  * Interface for a fresh Cookie instance with selected attribute(s)
  * only changed from the original instance.
  */
-interface CloneableCookieInterface extends CookieInterface
+interface CloneableInterface extends CookieInterface
```

Comment:
Please revert the name. `CloneableInterface` feels incomplete. Cloneable what? The current name is desired.

Reply:
Cloneable just like Countable Interface, But i agree it looks incomplete